backend/dataall/modules/dataset_sharing/services/share_managers/lf_share_manager.py

```diff
@@ -103,11 +103,14 @@ def build_shared_db_name(self) -> tuple:
         For shares after 2.3.0 the suffix returned is "_shared"
         :return: Shared database name, boolean indicating if it is a new share
         """
+        if self.source_database_name is None:
+            return '', True
```

Comment:
Returning '', True which will happen when the source database name is not set properly  which is likely if there are errors in setting catalog account details. The validation to check if the source database name , source account id, source region is done in the `process_revoked_shares` and `process_approved_shares` function